model/context.go

```diff
@@ -49,6 +54,9 @@ func LoadContext(taskId, buildId, versionId, patchId, projectId string) (Context
 		// Also lookup the ProjectRef itself and add it to context.
 		ctx.ProjectRef, err = FindOneProjectRef(projectId)
 		if err != nil {
+			if errors.Cause(err) == ErrProjectNotFound {
+				return ctx, errors.New("error finding the project")
```

Comment:
Probably should `return ctx, errors.Errorf("problem finding project '%s', projectId")` to include the projectId. Presumably it's obvious from the URL, but there may be cases where it might not be, and it gives a little extra info to the user that could aid in debugging.

Reply:
Thank you! I will edit it on Monday.